server/src/main/java/org/opensearch/common/document/serializer/DocumentFieldProtobufSerializer.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.document.serializer;
```

Comment:
Should we follow the same per-protocol package namings?

```suggestion
package org.opensearch.common.document.serializer.protobuf;
```